hw/ps4/liverpool/lvp_gfx.c

```diff
@@ -501,6 +502,10 @@ static uint32_t cp_handle_pm4_type3(
         cp_handle_pm4_it_wait_reg_mem(s, vmid, packet);
         break;
     }
+    // todo: This is a bit hacky for sending idle, but it at least takes care of letting orbis
+    // know for now, there also *should* be some mmio register checks that 'enable' this
+    // but until the emu progresses farther its tough to tell what is needed
+    liverpool_gc_ih_push_iv(s->ih, 0, IV_SRCID_UNK3_GUI_IDLE, 0);
```

Comment:
Is this related to the issue "GPU failed to become idle within 500ms" that shows up occasionally on UART logs?
I was pretty convinced that was some weird race that we lost due to issues with timers, but I'd be glad if it turns out to be just a missing interrupt. :-)

Reply:
Yea, it seems like it, and it at least seems to get rid of the message popping up every 500ms or so in uart. Just not sure on the exact conditions, checks, or if theres a better spot for it yet